lsp/server/tsconfig.json

```diff
@@ -11,6 +11,6 @@
         "composite": true
     },
 
-    "include": ["src"],
+    "include": ["src", "src/diagnostic/gql/misspelled-uiapi.ts"],
```

Comment:
Do we need to include `src/diagnostic/gql/misspelled-uiapi.ts`

Reply:
probably not, let me try removing.